state-chain/pallets/cf-flip/src/lib.rs

```diff
@@ -151,6 +210,9 @@ pub mod pallet {
 		SlashingRateUpdated {
 			slashing_rate: Permill,
 		},
+		FeeScalingRateUpdated {
+			fee_scaling_rate: FeeScalingRateConfig,
+		},
```

Comment:
Similar to the extrinsic, we could merge these into a single event `PalletConfigUpdated { update: PalletConfigUpdate }`

Reply:
this is consistent with the other pallets, though I agree, that makes sense